src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -142,7 +154,7 @@ const CourseEnrollments = ({ children }) => {
       <>
         {features.FEATURE_ENABLE_TOP_DOWN_ASSIGNMENT && (
           <CourseSection
-            title={COURSE_SECTION_TITLES.assigned}
+            title={isFirstVisit ? COURSE_SECTION_TITLES.firstTimeUserAndAssigned : COURSE_SECTION_TITLES.assigned}
```

Comment:
You need to make changes [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/6eb39df8e7984b0858483914a51d5003c576f62e/src/components/dashboard/main-content/course-enrollments/CourseSection.jsx#L36C3-L36C3) too. 
[Reason] I am saying this because we are showing a different design for courses count when we collapse the assigned course section.